stablehlo/reference/Ops.cpp

```diff
@@ -384,6 +384,16 @@ SmallVector<InterpreterValue> eval(Region &region,
       auto rhs = scope.findTensor(complexOp.getRhs());
       auto result = evalComplexOp(lhs, rhs, complexOp.getType());
       scope.add(complexOp.getResult(), result);
+    } else if (auto compositeOp = dyn_cast<CompositeOp>(op)) {
+      auto operands = scope.findTensors(compositeOp.getOperands());
+      SmallVector<InterpreterValue> values(operands.size());
+      std::transform(operands.begin(), operands.end(), values.begin(),
+                     [](const Tensor &t) { return InterpreterValue(t); });
+      auto func =
+          llvm::cast<func::FuncOp>(op.getParentOfType<ModuleOp>().lookupSymbol(
+              compositeOp.getDecomposition()));
+      auto results = eval(func.getBody(), values, fallback, process, parent);
```

Comment:
Not sure if we want to pass `parent` or `nullptr` for scope? I believe this would mean that the decomposition could refer to variables in the parent scope..which should be invalid.

Consider adding `evalCallOp` to do the "Create empty scope, lookup func, add results to parent scope", and just call it here. This would also be adding the ability for the interpreter to make calls, which is a nice feature that you've effectively implemented here.

Reply:
> scope

Good point! Done.

> `evalCallOp`

I want to keep this PR focused on just implementing the composite op. I agree this wold be a nice feature to add, but IMO that should go into its own PR.